scripts/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# build stage
+FROM archlinux/base AS builder
+
+# install build dependencies
+RUN pacman --noconfirm -Sy \
+    gcc \
+    git \
+    rust \
+    diffutils \
+    file \
+    awk \
+    pkg-config \
+    make
+
+WORKDIR /tmp/tox
+
+# download tox-node
+RUN git clone https://github.com/tox-rs/tox-node.git
+
+WORKDIR /tmp/tox/tox-node
+
+# build
+RUN cargo build --release
+
+# run stage
+FROM archlinux/base
+
+WORKDIR /usr/local/bin
```

Comment:
No need to cd into /usr/local, you can copy file using absolute path, so you can run it by absolute path too